skbio/diversity/alpha/base.py

```diff
@@ -71,11 +75,17 @@ def berger_parker_d(counts):
 
 
 def brillouin_d(counts):
-    """Calculate Brillouin index of alpha diversity, which is defined as:
+    """Calculate Brillouin index of alpha diversity.
+
+    This is calculated as follows:
 
     .. math::
 
-       HB = \\frac{\\ln N!-\\sum^5_{i=1}{\\ln n_i!}}{N}
+       HB = \\frac{\\ln N!-\\sum^s_{i=1}{\\ln n_i!}}{N}
```

Comment:
The equation and code should match. The code uses `gammaln(n+1)`, which is equivalent to `ln(abs(gamma(n+1)))`, where `gamma(n+1)` is equivalent to `n!` (see scipy docs [here](http://docs.scipy.org/doc/scipy-0.15.1/reference/generated/scipy.special.gammaln.html#scipy.special.gammaln) and [here](http://docs.scipy.org/doc/scipy-0.15.1/reference/generated/scipy.special.gamma.html#scipy.special.gamma) for details).